pkg/cmd/operator/operator.go

```diff
@@ -120,16 +112,12 @@ func Run(healthPort, monitoringPort int32, leaderElection bool, leaderElectionID
 		}
 	}
 
-	logLevelEnabler := loglvl{
-		Level: logLevel,
-	}
-
-	logf.SetLogger(zap.New(func(o *zap.Options) {
+	logf.SetLogger(zapctrl.New(func(o *zapctrl.Options) {
 		o.Development = false
-		o.Level = logLevelEnabler
+		o.Level = zap.NewAtomicLevelAt(logLevel)
```

Comment:
Am sure that this works and is better but could you just explain how. Not least might be good to stick a comment about it to give anyonelse an idea seeing as we both got confused originally.

Reply:
I saw that zap is doing it this way when adding the default level options so I thought it must be the right way to do it 😄

Reply:
Have added some comments

Reply:
@phantomjinx can you have another look, please